tasks/templates/00-default/justfile

```diff
@@ -48,3 +48,17 @@ execute hdPath='0':
     --sig "runJson(string,bytes)" "${location}/${bundle}.json" ${SIGNATURES} \
     --ledger --hd-paths "m/44'/60'/{{hdPath}}'/0/0" --broadcast \
     --sender ${sender}
+
+add-transaction to sig *params:
+  #!/usr/bin/env bash
+  cd ../..
+  newbundle="${bundle}-new"
+  backupbundle="${bundle}-$(date -u '+%Y-%m-%d_%H-%M-%S')"
+  DATA=$(cast calldata '{{sig}}' {{params}})
+  jq --arg to "{{to}}" --arg data "${DATA}" \
+    '.transactions += [{"to": $to, "data": $data}]' \
+    "${location}/${bundle}.json" \
+    > "${location}/${newbundle}.json"
+  mv "${location}/${bundle}.json" "${location}/${backupbundle}.json" 
+  mv "${location}/${newbundle}.json" "${location}/${bundle}.json"
+  echo "Old bundle backed up to ${backupbundle}.json"
```

Comment:
do you think we need the back up stuff? Feels like git should handle that in most cases.

Reply:
yeah was thinking the same. But if you add two in a row and you bork the 2nd, you'd be happy for the backup 😅 I'm ok with either.

Reply:
It's your feature, let's keep it.